### PR TITLE
!: edit page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/!.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 `!{{number}}`
 
-- Substitute with a command that was used specified lines back:
+- Substitute with a command that was used specified number of lines back:
```
Comment (kbdharun) [inline pages/linux/!.md:14]: Thanks for your contribution.

How about this?

```suggestion
- Substitute with a command that uses a specified number of lines back:
```

Comment (Managor) [reply pages/linux/!.md:14]: I feel like that's not grammatically correct.

Comment (mfrw) [inline pages/linux/!.md:14]: Well maybe this:
```suggestion
- Substitute with a command that is "number" in history:
```

Comment (pixelcmtd) [reply pages/linux/!.md:14]: I'm pretty sure it's grammatically correct, but I'd interpret it to mean something different.

My suggestions would be:

```suggestion
- Substitute with a command that was used a specified number of lines back:
```

```suggestion
- Substitute with a command from a specified number of lines back:
```

**Comment by pixelcmtd** [comment]
Btw, why is this page in `linux`?

**Review by pixelcmtd — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Comment by Managor** [comment]
I default to Linux when I don't know if it's available on other platforms or not.

**Comment by pixelcmtd** [comment]
> I default to Linux when I don't know if it's available on other platforms or not.

The `More information` cites `bash` and I don't think that has any platform-specific behavior (at least of such a magnitude)

**Pull request merged**